### PR TITLE
bug: only get umip number from contentful or json table

### PR DESCRIPTION
**Pull request by ryanwolhuter**
### Summary

There was some incorrect logic for getting umip numbers and urls for votes. I've removed that function which made wrong assumptions about the relationship between admin numbers and umip numbers.

See comments in the PR for the specific details.

### CODE REVIEW

File: helpers/voting/getVoteMetaData.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ export function getVoteMetaData(
       umipDataFromContentful?.description ??
       "No description was found for this UMIP.";
     const umipUrl = umipDataFromContentful?.umipLink;
-    const umipNumber = getUmipNumber(decodedIdentifier);
-    const links = makeVoteLinks(transactionHash, umipNumber);
+    const umipNumber = umipDataFromContentful?.number;
```
Comment (ryanwolhuter) [inline helpers/voting/getVoteMetaData.ts:30]: For UMA governance, we only trust the information that comes from contentful. That data has the umip number and the url for the umip on github.

File: helpers/voting/getVoteMetaData.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ export function getVoteMetaData(
     const umipUrl =
       "https://github.com/UMAprotocol/UMIPs/blob/448375e1b9d2bd24dfd0627805ef6a7c2d72f74f/UMIPs/umip-157.md";
     const umipNumber = 157;
-    const links = makeVoteLinks(transactionHash, umipNumber);
+    const links = makeVoteLinks(transactionHash, umipNumber, umipUrl);
```
Comment (ryanwolhuter) [inline helpers/voting/getVoteMetaData.ts:54]: For across requests, the umip number and url are always the same

File: helpers/voting/getVoteMetaData.ts [hunk 3]
```diff
@@ -106,8 +106,8 @@ export function getVoteMetaData(
     const title = identifierDetails.identifier;
     const description = identifierDetails.summary;
     const umipUrl = identifierDetails.umipLink.url;
-    const umipNumber = getUmipNumber(identifierDetails.umipLink.number);
-    const links = makeVoteLinks(transactionHash, umipNumber);
+    const umipNumber = Number(identifierDetails.umipLink.number);
```
Comment (ryanwolhuter) [inline helpers/voting/getVoteMetaData.ts:109]: For known identifiers, the umip number and url are found in the json table.

File: helpers/voting/getVoteMetaData.ts [hunk 4]
```diff
@@ -178,25 +178,18 @@ function getDescriptionFromAncillaryData(
   );
 }
 
-function getUmipNumber(umipOrAdmin: string | undefined) {
```
Comment (ryanwolhuter) [inline helpers/voting/getVoteMetaData.ts:181]: This was always wrong and should never have been included.

**Review by daywiss — APPROVED** [review]

**Comment by daywiss** [comment]
just a nit but can you change the title of your prs to "fix:" instead of "bug:" following conventional commits

**Pull request merged**